app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -55,8 +55,7 @@ def auto_attach
     api :GET, "/hosts/:host_id/subscriptions/events", N_("List subscription events for the host")
     param :host_id, Integer, :desc => N_("Id of the host"), :required => true
     def events
-      collection = full_result_response(@host.subscription_facet.candlepin_consumer.events.map { |e| OpenStruct.new(e) })
-      respond_for_index :collection => collection
+      respond_for_index :collection => []
```

Comment:
Let's add a deprecation warning to this API. Check out our usage of `::Foreman::Deprecation.api_deprecation_warning` in some controllers. Since this code will land in 3.14 we can mention that the API will be removed in 3.16